hollow/src/main/java/com/netflix/hollow/api/consumer/HollowConsumer.java

```diff
@@ -231,6 +233,10 @@ public void triggerRefresh() {
         }
     }
 
+    public void fireAnnouncementDetected(long newVersion, Map<String, String> metadata, boolean isPinned) {
```

Comment:
Just FYI - You might want to clean up this PR as it has an additional unrelated commit.